src/wp-content/themes/twentytwentyone/block-templates/default.html

```diff
@@ -0,0 +1,44 @@
+<!-- wp:group {"align":"full","tagName":"header"} -->
+<header class="wp-block-group alignfull"><!-- wp:spacer {"height":46} -->
+<div style="height:46px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:site-title {"textAlign":"center"} /-->
+
+<!-- wp:site-tagline {"textAlign":"center","fontSize":"small"} /-->
+
+<!-- wp:spacer {"height":57} -->
+<div style="height:57px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+</header>
+<!-- /wp:group -->
+
+<!-- wp:group {"align":"full","tagName":"main"} -->
+<div class="wp-block-group alignfull entry-header"><!-- wp:group {"align":"wide","layout":{"inherit":true}} -->
```

Comment:
I think the entry-header class should go on the header tag right above. This is adding a weird padding and border at the bottom of the page

Reply:
Great catch, thank you! I've taken a different approach to this in https://github.com/WordPress/wordpress-develop/pull/1455/commits/3bbcd26e27f0d2d20fe6867ebf7dedc9e3db7756, and it should be working better now. 👍